examples/misc/guestbook/5-guestbook-controller.json

```diff
@@ -21,7 +21,7 @@
             "spec":{
                 "containers":[{
                     "name":"guestbook",
-                    "image":"gcr.io/google-samples/gb-frontend:v6",
+                    "image":"us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5",
```

Comment:
Same comment as in https://github.com/cilium/cilium/pull/29603#discussion_r1414021621

Apart from changing the registry, the image is downgraded from v6 to v5. I assume it wasn't intentional?